Bottom: 7101ae0e3ef1dafceb1ebf51c9e23ec4a8fc9d32
Top:    e205fffedab565c740b2688400ca272317ab89b7
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-22 11:28:23 -0400

Support for AC97 on Phytec pmc030 base board.

Support for AC97 on Phytec pmc030 base board. This uses the generic fabric driver binding.

Signed-off-by: Jon Smirl <jonsmirl@gmail.com>


---

diff --git a/arch/powerpc/boot/dts/pcm030.dts b/arch/powerpc/boot/dts/pcm030.dts
index 8958347..001c8ef 100644
--- a/arch/powerpc/boot/dts/pcm030.dts
+++ b/arch/powerpc/boot/dts/pcm030.dts
@@ -190,6 +190,14 @@
 			cell-index = <0>;
 			reg = <0x2000 0x100>;
 			interrupts = <2 1 0>;
+
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			codec@0 {
+				compatible = "wlf,wm9712";
+				reg = <0x0>;
+			};
 		};
 
 		/* PSC2 port is used by CAN1/2 */
diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index a16c7d6..18a19bf 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -48,4 +48,11 @@ config SND_MPC52xx_SOC_EFIKA
 	help
 	  Say Y if you want to add support for sound on the Efika.
 
+config SND_MPC52xx_SOC_PCM030
+	tristate "SoC AC97 Audio support for Phytec pcm030 and WM9712"
+	depends on PPC_MPC5200_SIMPLE
+	select SND_SOC_MPC5200_AC97
+	select SND_SOC_WM9712
+	help
+	  Say Y if you want to add support for sound on the Phytec pcm030 baseboard.
